src/Modules/Factory/ModuleFactoryImpl1.cc

```diff
@@ -193,6 +195,7 @@ void ModuleDescriptionLookup::addEssentialModules()
   addModuleDesc<ConvertIndicesToFieldData>("Real ported module", "..."); 
   addModuleDesc<SolveInverseProblemWithTikhonov>("...", "...");
   addModuleDesc<ShowColorMapModule>("ShowColorMap", "Visualization", "SCIRun", "Real ported module", "..."); 
+  addModuleDesc<ConvertMatrixToString>("ConvertMatrixToString","Converters","SCIRun","...","...");
```

Comment:
@mhansen1 This is an acceptable factory diff. Although you'll need to merge again, with your own change from the other PR, and with master, so be careful.